### PR TITLE
use fix from admin page for select on main site

### PR DESCRIPTION
**Pull request by billyeh**
I found this fix on this page already https://github.com/attendanceproject/djattendance/blob/dev/ap/templates/admin/base_site.html but that's for the admin site.

Without this, the select2 box for e.g. on the Create Event page for trainees doesn't work after a submission or refresh. Seems like the scripts have to be re-included to re-initialize the select2 box.

### CODE REVIEW

**Comment by billyeh** [comment]
Actually, I wonder we should wait for @lifedispenser's react reorg

**Comment by radicalray** [comment]
His react reorg should not affect admin too much.

**Comment by billyeh** [comment]
Hmm I actually wanted to move these lines of code into the non-admin site from the admin site, so I was thinking it'd be affected by his reorg.

**Comment by billyeh** [comment]
So yeah this is already documented on #154

File: ap/templates/base.html [hunk 1]
```diff
@@ -20,6 +20,9 @@
       {% load bootstrap3 %}
 
       <link href="{% static "css/base.css" %}" rel="stylesheet">
+      {# Quick fix b/x select2 css refuses to load after 2nd page refresh #}
+      <link rel="stylesheet" type="text/css" href="{% static 'django_select2/css/select2.css' %}" media="all">
```
Comment (billyeh) [inline ap/templates/base.html]: I want to add these scripts to the announcements creation and update pages that I'm working on. You're right, that makes a lot more sense. Thanks!

**Pull request closed without merging**